src/mame/includes/spectrum.h

```diff
@@ -216,13 +218,19 @@ class spectrum_128_state : public spectrum_state
 {
 public:
 	spectrum_128_state(const machine_config &mconfig, device_type type, const char *tag) :
-		spectrum_state(mconfig, type, tag)
+		spectrum_state(mconfig, type, tag),
+		m_bank_rom(*this, "bank_rom%u", 0U),
+		m_bank_ram(*this, "bank_ram%u", 0U)
 		{ }
```

Comment:
Can you please outdent the empty braces to the level of the constructor name while you’re editing this part of the file?